release/test_tag_release.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+source _test_util.sh
+source release_gold.sh --source-only
```

Comment:
@lucasmiranda0, please create a new file named `_github.sh` and move the function `invoke_github_api` to this new file and then import it here like: `source _github.sh`.

Your beloved and handsome friend @kiwm already did this previously in another pull, please reach out him. :wink: